CHANGELOG.md

```diff
@@ -6,6 +6,112 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 <!--
 TODO: Insert version codename, and username of the contributor that named the release.
 -->
+## [23.02rc1] - 2023-02-08
+
+### Added
+
+ - Plugins: `sql` plugin command to perform server-side complex queries. ([#5679])
+ - JSON-RPC: new command `listpeerchannels` now contains information on direct channels with our peers. ([#5825])
+ - RPC `listpeers` output now has `num_channels`. ([#5968])
```

Comment:
This should go lower, since it's a detail, and be more descriptive:
```
JSON-RPC: `listpeers` output now has `num_channels` as `channels` is deprecated (see `listpeerchannels`).
```